sklego/decomposition/pca_reconstruction.py

```diff
@@ -93,13 +93,9 @@ def fit(self, X, y=None):
         )
         self.pca_.fit(X, y)
         self.offset_ = -self.threshold
-        return self
 
-    def transform(self, X):
```

Comment:
This method was throwing off the checks as they dynamically check for method names